common/ch_channel_map/any_series

```diff
@@ -3,9 +3,15 @@ declare -A RELNAME_MAP=(
     [antelope]=2023.1
     [bobcat]=2023.2
     [caracal]=2024.1
+    [dalmatian]=latest
```

Comment:
dalmatian is 2024.2

Reply:
at present the only way to deploy noble (to get D and E uca) is to use latest/edge charms